### PR TITLE
Use `%q` instead of `%s` for strings with quotes

### PR DESCRIPTION
**Pull request by MustafaSaber**
follow up on https://github.com/zalando/skipper/pull/2691

### CODE REVIEW

File: cmd/webhook/admission/admission_test.go [hunk 1]
```diff
@@ -125,12 +125,12 @@ func TestRouteGroupAdmitter(t *testing.T) {
...
 			name:      "invalid routgroup multiple filters per json/yaml array item",
 			inputFile: "rg-with-multiple-filters.json",
-			message:   `single filter expected at \"status(201) -> inlineContent(\"hi\")\"\nsingle filter expected at \" \"`,
+			message:   `single filter expected at \"status(201) -> inlineContent(\\\"hi\\\")\"\nsingle filter expected at \" \"`,
 		},
 		{
 			name:      "invalid routgroup multiple predicates per json/yaml array item",
 			inputFile: "rg-with-multiple-predicates.json",
-			message:   `single predicate expected at \"Method(\"GET\") && Path(\"/\")\"\nsingle predicate expected at \" \"`,
+			message:   `single predicate expected at \"Method(\\\"GET\\\") && Path(\\\"/\\\")\"\nsingle predicate expected at \" \"`,
```
Comment (szuecs) [inline cmd/webhook/admission/admission_test.go:133]: Why do you need to quote this?

Comment (MustafaSaber) [reply cmd/webhook/admission/admission_test.go:133]: Nothing specific, was just trying to make it more readable by quoting

Comment (szuecs) [reply cmd/webhook/admission/admission_test.go:133]: If you use backticks then you don't need to quote `"` anymore and %q will add `"` you strings.

Comment (MustafaSaber) [reply cmd/webhook/admission/admission_test.go:133]: this is `%q` quotes

Comment (szuecs) [reply cmd/webhook/admission/admission_test.go:133]: true

**Comment by szuecs** [comment]
:+1:

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**